Apps/WebClient/src/ClientApp/src/store/modules/auth/actions.ts

```diff
@@ -125,9 +145,15 @@ export const actions: AuthActions = {
             });
     },
     signOutOidc(): void {
+        const authService: IAuthenticationService = container.get<IAuthenticationService>(
```

Comment:
why is it necessary to move it inside the methods? it looks redundant to me!

Reply:
That is necessary to avoid the dependency being solved at inclusion time by the import. We could make this a class so that it has members that it can use, but otherwise, the definition needs to per method